samples/webview/webview.cpp

```diff
@@ -408,6 +410,7 @@ WebFrame::WebFrame(const wxString& url) :
     m_tools_medium = m_tools_menu->AppendCheckItem(wxID_ANY, _("Medium"));
     m_tools_large = m_tools_menu->AppendCheckItem(wxID_ANY, _("Large"));
     m_tools_largest = m_tools_menu->AppendCheckItem(wxID_ANY, _("Largest"));
+    m_tools_custom = m_tools_menu->AppendCheckItem(wxID_ANY, _("Custom Size"));
```

Comment:
I have no idea why do we use check items here instead of radio items... This would have been simpler, as we wouldn't need to manually uncheck them.

Again, this is not something that prevents this PR from being merged, but if you'd like to simplify it, please feel free to do it.